src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { Routes, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Navbar from './components/Navbar';
+import BookLists from './components/BookList';
+import Categories from './components/CategoriesPage';
+import Store from './redux/ConfigureStore';
```

Comment:
- [ ] Kindly make the `ConfigureStore` => `configureStore` because your file name has a small **c** you can cross check it and it is causing an issue and breaking your application :thinking:

Reply:
DONE

Reply:
![Screenshot (491)](https://user-images.githubusercontent.com/96092850/192876583-330fa7b7-51dd-4e21-aedb-9cb9c6589cba.jpg)

I have done it. thanks.